public/js/io/model.js

```diff
@@ -140,13 +140,23 @@ angular.module('app').service('ioModel', function (connection, $q) {
     this.importBundle = function (bundle, datasourceRef) {
         const additions = [];
         const messages = [];
+        const replace = [];
 
         const layerPromises = [];
         for (const layer of bundle.layers) {
             const p = getLayer(layer._id).then(l => {
                 if (l) {
-                    messages.push('Layer was not imported because it already exists in database: ' + l.name);
-                    return;
+                    if (layer.replace === true) {
+                        var msg = gettextCatalog.getString('This layer was replaced:' + ' ');
```

Comment:
This doesn't work.
This will be extracted in the PO file as "This layer was replaced:" (OK) but during execution it will search for "This layer was replaced: " (with a trailing space) and it will fail to find a translation